---
float32 current_position
bool released
---
float32 current_position
float32 current_velocity
float32 motor_current
